### DISCUSSION TITLE
Unable to reverify academic affiliation because in Erasmus

### DISCUSSION THREAD

**Discussion by xplosionmind** [community/community#41220] [GitHub Education]
I am spending one semester abroad for the Erasmus project, and therefore I am blocked in my process of re-verification of my academic affiliation, since I am located in another country. Is there a way to bypass this automatic block?

Thanks a lot.

**Comment by samus-aran** [comment]
Hi @xplosionmind 👋 

We're happy you're here! We do support remote learning but as you can imagine there are many types of academic verification and institutes around the world and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient with it.

Here are a few tips:

1. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Dated Student ID being at the top. If you don't have a Student ID do try and acquire evidence that is positioned higher rather than lower in that option list.
2. Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted.
3. The verification process is always progressing so it may be different than it was when previously verified. If you are required to submit additional evidence please try and acquire it from your school as it will help your application be more likely successful.

Let us know how you get on 👍

**Reply by xplosionmind** [reply]
Thanks for your reply! Unfortunately, as you can see from the screenshot, I do not see any verification alternatives. The evidence is consistent across my GitHub credentials. I really have no time to involve my university in this. Isn’t there a way to just manually verify me?

![image](https://user-images.githubusercontent.com/42862428/208056712-cd8fc272-8aed-4df9-aafd-7edc358726c8.png)

**Reply by samus-aran** [reply]
Thanks for your screenshot. 👍

We are unable to provide manual reviews of individual applications and recommend to adjust your application following the rejection reasons.

Please do ensure your evidence corresponds to details in your GitHub profile. Does the name you use in your profile exist on your evidence exactly or do you use a variation/nickname? Like if I had the name "Andy" but used the nickname "Andi" on my profile they would not match. It's these small, but important points, to consider when applying. Compare your evidence to your profile information and update accordingly and re-apply.

Let us know how you get on.
